test/options.jl

```diff
@@ -21,8 +21,8 @@ m = MathProgBase.ConicModel(s)
 MathProgBase.loadproblem!(m, -obj, A, rowub, [(:NonNeg,1:3)],[(:NonNeg,1:5)])
 MathProgBase.optimize!(m)
 
-@test isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-9, rtol=0.0)
-@test !isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-10, rtol=0.0)
+@test isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-7, rtol=0.0)
+@test !isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-8, rtol=0.0)
```

Comment:
How about removing this one as well?

Reply:
Very well! I left only one lower bounds test with a comment.